### PR TITLE
fix: Button behind keyboard

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Bugfix

This PR is the follow up of https://github.com/artsy/eigen/pull/6391

The button should stay behind the keyboard as per the discussion [here](https://artsyproduct.atlassian.net/browse/CX-2417?focusedCommentId=46885). 

Before

https://user-images.githubusercontent.com/17421923/157662263-32751a3b-350c-4540-a625-5d1ca412d424.mov

After

https://user-images.githubusercontent.com/17421923/157662234-e8aef3a0-c7fa-481b-ab04-3f8625514bf1.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Button should stay behind the keyboard on MyCollection Add/Edit artwork form - Sam

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- 

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Button should stay behind the keyboard on MyCollection Add/Edit artwork form - Sam)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f35d33cfc70dbf36e3f2ce7b5fb239fd215fdf33
</p>

File: src/app/Scenes/MyCollection/Screens/ArtworkForm/Screens/MyCollectionArtworkFormMain.tsx [hunk 1]
```diff
@@ -76,152 +76,154 @@ export const MyCollectionArtworkFormMain: React.FC<
...
+          {/* Show validation errors during development */}
+          {!!(SHOW_FORM_VALIDATION_ERRORS_IN_DEV && __DEV__ && formik.errors) && (
+            <ScreenMargin>
+              <Box my={2}>
+                <Sans size="3">Errors: {JSON.stringify(formik.errors)}</Sans>
+              </Box>
+            </ScreenMargin>
+          )}
+        </ScrollView>
+      </ArtsyKeyboardAvoidingView>
```
Comment (rajsam003) [inline src/app/Scenes/MyCollection/Screens/ArtworkForm/Screens/MyCollectionArtworkFormMain.tsx:226]: I dont know why it shows this big difference. The change is only this. 

image

Comment (rajsam003) [reply src/app/Scenes/MyCollection/Screens/ArtworkForm/Screens/MyCollectionArtworkFormMain.tsx:226]: VSCode is smarter than Github 👮‍♂️

**Review by dariakoko — APPROVED** [review]
Looks good! 
Thanks for the screenshot with the diff, saves time 🚀

**Pull request merged**